base/compiler/ssair/ir.jl

```diff
@@ -1195,7 +1195,6 @@ function fixup_node(compact::IncrementalCompact, @nospecialize(stmt))
         return compact.ssa_rename[stmt.id]
     else
         urs = userefs(stmt)
```

Comment:
@Keno AFAICT `userefs` never returns `()`. Is that right?

Reply:
I believe that replaced by the `.relevant` flag in the UseRefIterator for type stability.